Projects/UOContent/Engines/Spawners/EditSpawnCommand.cs

```diff
@@ -67,45 +68,65 @@ public override void ExecuteList(CommandEventArgs e, List<object> list)
 
             var argSpan = e.ArgString.AsSpan(name.Length + 1);
             var setIndex = argSpan.InsensitiveIndexOf("set ");
+            var where = argSpan.InsensitiveIndexOf("match ");
+           
+            ReadOnlySpan<char> props = null, findmatch = null;
 
-            ReadOnlySpan<char> props = null;
-
-            if (setIndex > -1)
+            if (setIndex > -1 || where > -1)
             {
                 var start = setIndex + 4;
-                props = argSpan.Slice(start, argSpan.Length - start);
-                argSpan = argSpan.SliceToLength(setIndex);
+                var len = where>-1? where : argSpan.Length;
+                findmatch = argSpan.Slice(len < argSpan.Length ? len + 6 : argSpan.Length);
+                if (setIndex>-1)
+                {
+                    props = argSpan.Slice(start, len - start);
+                    argSpan = argSpan.SliceToLength(setIndex);
+                }
+                else
+                {
+                    argSpan = argSpan.SliceToLength(argSpan.Length-len);
+                }
             }
 
             var argStr = argSpan.ToString().DefaultIfNullOrEmpty(null);
             var propsStr = props.ToString().DefaultIfNullOrEmpty(null);
+            var whereStr = findmatch.ToString().DefaultIfNullOrEmpty(null);
```

Comment:
Note to myself: I should make a `Span<char>` version of `DefaultIfNullOrEmpty` so that we do not need to allocate  if span is null or empty.
To do this, need to look into a way to find `whitespace` using `Span<char>`. 🤔